pages/home/_cards.js

```diff
@@ -13,25 +19,38 @@ const SpeakerContent = styled.div`
   justify-content: center;
   align-items: center;
   background-color: var(--color-orange);
-  width: 37rem;
+  width: 100%;
+  max-width: 370px;
   padding: 2.4rem;
   border-radius: 2.5rem;
+  margin-top: -51px;
+  z-index: 1;
+`
+const StyledImage = styled(Image)`
+  border-radius: 2.5rem;
 `
 
-const Cards = () => {
+const Cards = (props) => {
+  const { imageSource, name, children } = props
```

Comment:
```suggestion
  const { src, name, children } = props
```

Reply:
We should try to keep the props naming consistent to what it gets passed to.

Reply:
The reason why I didn't use the same name was that I wasn't sure if it was good practice to name variables the same as native html parameters. Will change if this is fine.